src/ucp/api/ucp.h

```diff
@@ -2791,6 +2791,24 @@ ucs_status_t ucp_tag_recv_request_test(void *request, ucp_tag_recv_info_t *info)
 
 /**
  * @ingroup UCP_COMM
+ * @brief Check the status and currently available state of non-blocking request
+ *        returned from @ref ucp_stream_recv_nb routine.
```

Comment:
we do not use such indentation for doxygen comments

Reply:
see [here](https://github.com/openucx/ucx/blob/master/src/ucp/api/ucp.h#L2084), [here](https://github.com/openucx/ucx/blob/master/src/ucp/api/ucp.h#L2775) and many other places also with `@return` and `@note` multiline labels.

Reply:
ok, I've seen other example, but the concern was also about the detailed description below. You keep indentation there as well, which looks weird

Reply:
Just copied/pasted from `tag`.

Reply:
I mean this indentation.
```
+ * This routine checks the state and returns current status of the request
 + *      returned from @ref ucp_stream_recv_nb routine. Any value different from
 + *      UCS_INPROGRESS means that the request is in a completed state.
```

Reply:
Fixed in both places